tests/server/test_query_params.py

```diff
@@ -386,12 +395,23 @@ def test_list_length(self):
         # self._check_response(request, expected_ids, len(expected_ids))
 
     def test_list_has_only(self):
+        """ Test HAS ONLY query on elements.
+
+        Curiously this test fails under mongomock when $size is 1, but works with a real mongo.
+
+        The queries produced in each case should be:
+            - `{"elements": {"$all": ["Ac", "Mg"], "$size": 2}}`
+            - `{"elements": {"$all": ["Ac"], "$size": 1}}`
+
```

Comment:
I think you should add a link to your PR and a comment or TODO that all this text should be removed when we start depending on the next `mongomock` version.

Reply:
Good idea, done.